src/files/fileset.rs

```diff
@@ -204,333 +286,101 @@ impl LoadedMod {
 
 #[derive(Debug)]
 pub struct Fileset {
-    /// The CK3 game directory.
-    vanilla_root: Option<PathBuf>,
-
-    /// Extra CK3 directory loaded before vanilla.
-    #[cfg(feature = "jomini")]
-    clausewitz_root: Option<PathBuf>,
-
-    /// Extra CK3 directory loaded before vanilla.
-    #[cfg(feature = "jomini")]
-    jomini_root: Option<PathBuf>,
+    #[allow(dead_code)]
+    pub(super) paths: FilePaths,
 
     /// The mod being analyzed.
-    the_mod: LoadedMod,
+    #[allow(dead_code)]
```

Comment:
Why these three dead_code allows? (the_mod, loaded_dlcs, and db). Surely they're used for all games.